src/main/java/org/folio/ld/fingerprint/service/FingerprintServiceImpl.java

```diff
@@ -49,12 +49,16 @@ private List<List<String>> getFingerprint(Resource resource, FingerprintRule rul
     ofNullable(rule).flatMap(r -> collectLabel(resource, r.label())).ifPresent(fingerprint::add);
     fingerprint.addAll(collectProperties(resource, rule));
     fingerprint.addAll(collectEdges(resource, rule));
-    return fingerprint;
+    return fingerprint.stream()
+      .sorted(comparing((List<String> pair) -> pair.get(0)).thenComparing(pair -> pair.get(1)))
```

Comment:
Not required by current task directly, but there is idea for refactoring:

That's not really clear why it's a pair here. 
We have to read a history of fingerprint variable including inner methods to see that there is a lists of two resources inside.

To make it clear we should use a Pair record instead in every method returning List of two elements in this class.
Something like:
record Pair(String key, String value) {}

Reply:
done

Reply:
@PBobylev FYI, because of this refactoring we have to make some changes in the modules that use this module. See the example in the ```ObjectMapperConfig``` in the following PR https://github.com/FOLIO-EIS/mod-linked-data/pull/306

Reply:
I've added one more change to avoid modification of ```ObjectMapperConfig```s in other modules.